vmngclient/tests/test_configuration_group_endpoint.py

```diff
@@ -0,0 +1,171 @@
+import unittest
```

Comment:
Because endpoint layer is purely declarative. We don't see benefit of unittest endpoints (methods have no body, it is expected it sends model correctly as decorators are fully unittested: /vmngclient/tests/test_endpoints.py)
If the model is incompatible with we won't detect it in unittest anyways.

Reply:
So, should I remove this file from commit?

Reply:
yes